src/OrchardCore.Modules/OrchardCore.Facebook/Login/Configuration/FacebookLoginConfiguration.cs

```diff
@@ -43,6 +43,11 @@ public void Configure(AuthenticationOptions options)
                 return;
             }
 
+            if (string.IsNullOrWhiteSpace(_facebookSettings.AppId) || string.IsNullOrWhiteSpace(_facebookSettings.AppSecret))
+            {
+                return;
```

Comment:
Here you can add a log with something like.

The Facebook login provider is enabled but not configured.